src/modules/users/users.service.ts

```diff
@@ -503,12 +510,61 @@ export class UsersService {
     try {
       const hashedPassword = await this.hashPassword(password);
 
+      const user = await this.getUserById(id);
```

Comment:
again from my understanding we validate it in step above, no?

Reply:
I agree, I deleted the if statement, but I need the user to get his email and name to send an email notification about password change